_posts/en/newsletters/2019-04-30-newsletter.md

```diff
@@ -0,0 +1,203 @@
+  of storage or memory. An additional reason given is to provide some closure to
+  proceed if a payment is not initiated or completed rather than leave
+  it dangling. David Harding points out that traditional businesses put
+  expiration dates on invoices to avoid an obligation to deliver goods in the
+  future at a previously offered price.
+
+- [Are there still miners or mining pools which refuse to implement
+  SegWit?]({{bse}}86208) Mark Erhardt provides extensive analysis demonstrating
+  that essentially the answer is "no." Only 0.03% non-empty blocks from
+  the past year had no SegWit transactions, and the two primary miners of
+  those few blocks have demonstrated in 2019 that they’re mining blocks
+  with SegWit transactions.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[LND][lnd repo], [C-Lightning][c-lightning repo], [Eclair][eclair repo],
+[libsecp256k1][libsecp256k1 repo], and [Bitcoin Improvement Proposals
+(BIPs)][bips repo].  Note: unless otherwise noted, all merges described
+for Bitcoin Core are to its master development branch; some may also be
+backported to its pending release.*
+
+- [Bitcoin Core #14039][] causes Bitcoin Core to reject transactions
+  that use the segwit-style extended transaction encoding when they
+  don't include any segwit inputs.  The extended transaction encoding
+  includes a segwit marker, a segwit flag, and witness data fields.
+  Signatures included in legacy inputs don't commit to these fields, so
+  adding the fields to a transaction produces a (small) waste of
+  bandwidth in a transaction consisting entirely of legacy inputs.  For
+  this reason, [BIP144][] specified that non-segwit transactions should
+  use the legacy transaction format.  Previously Bitcoin Core accepted
+  these misformated transactions and normalized them by stripping out
+  the segwit-only parts before calculating their size (weight) or
+  relaying them to other peers; now it will refuse to accept
+  transactions that don't use the correct format.
+
+- [Bitcoin Core #15846][] updates the node to accept transactions into
+  its mempool for relay and mining if any outputs in the transaction pay
+  segwit address versions 1 through 16---the versions reserved for
+  future protocol upgrades.  Previously, such transaction would be
+  rejected.  Any money sent to a future version address is not secure
+  (any miner can spend it) until users enforce a soft fork giving that
+  address special meaning, similar to the special meaning of segwit
+  version 0 addresses for P2WPKH and P2WSH.  That means no one should be
+  using version 1+ addresses today.  However, if anyone does create such
+  an address and asks a segwit-supporting wallet or service to pay it,
+  this change ensures that the transaction will be relayed and mined
+  like any other transaction.  (In an already-written future edition of
```

Comment:
I think you can drop the 'already-written' bit. Just say "A future edition of this newsletter's *bech32 sending support* section will go ..."